### PR TITLE
Add useage examples to DARS guide

### LINKED ISSUE #793: Update DARS guide with how-to examples
Current version of the Data Analytics Reference Stack guide only shows how to install.  Would be a more useful document with examples for how to use and how to interact with the other stacks offered by Clear Linux.

### PR DESCRIPTION
**Pull request by bd-dean**
Adds PySpark examples and points reader to the stacks/usecases repository for walkthrough with Scala. Also cleans up & reorders use/build Docker image section.   Closes #793

### CODE REVIEW

File: source/guides/stacks/dars.rst [hunk 1]
```diff
@@ -41,78 +42,120 @@ in the |CL|-based containers may not be the latest released by |CL|.
...
+
 #. Once you have downloaded the image, you can run it with
 
    .. code-block:: bash
 
       docker run -it --ulimit nofile=1000000:1000000 --name mkl <name of image>
 
-   This will launch the image and drop you into a bash shell inside the
-   container.  You will see output similar to the following:
+   This will launch the image and drop you into a bash shell inside the container. The :command:`--ulimit nofile=` parameter is required in order to increase the allowed number of open files for the Apache Spark engine.
```
Comment (mvincerx) [inline source/guides/stacks/dars.rst]: One question on `docker run` .  For `<name of image>`, do you expect users to run a `docker ps` to find the name of the image first?

File: source/guides/stacks/dars.rst [hunk 2]
```diff
@@ -41,78 +42,120 @@ in the |CL|-based containers may not be the latest released by |CL|.
...
-   .. code-block:: console
+   .. code-block:: bash
+
+      git clone https://github.com/clearlinux/dockerfiles.git
+
+#. Inside the stacks/dars/mkl directory, use docker with the :file:`Dockerfile` to build the  MKL image.
+
+   .. code-block:: bash
+
+      docker build --no-cache -t clearlinux/stacks-dars-mkl .
```
Comment (mvincerx) [inline source/guides/stacks/dars.rst]: Add a step to specify the directory where to run command.

File: source/guides/stacks/dars.rst [hunk 3]
```diff
@@ -41,78 +42,120 @@ in the |CL|-based containers may not be the latest released by |CL|.
...
-   This will launch the image and drop you into a bash shell inside the
-   container.  You will see output similar to the following:
+   This will launch the image and drop you into a bash shell inside the container. The :command:`--ulimit nofile=` parameter is required in order to increase the allowed number of open files for the Apache Spark engine.
+
+
+Building DARS images
+====================
+
+If you choose to build your own DARS container images, you can customize them as needed. Use the :file:`Dockerfile` included in the Github\* repository as your baseline.
+
```
Comment (mvincerx) [inline source/guides/stacks/dars.rst]:
```suggestion
You can also follow `custom-app-container`
```

File: source/guides/stacks/dars.rst (hunk 1 above)
Comment (bd-dean) [reply source/guides/stacks/dars.rst]: added -- good question.

File: source/guides/stacks/dars.rst (hunk 2 above)
Comment (bd-dean) [reply source/guides/stacks/dars.rst]: Added.

File: source/guides/stacks/dars.rst (hunk 3 above)
Comment (bd-dean) [reply source/guides/stacks/dars.rst]: added.

File: source/guides/stacks/dars.rst [hunk 4]
```diff
@@ -41,78 +42,137 @@ in the |CL|-based containers may not be the latest released by |CL|.
...
-   increase the number of open files opened at certain point by the spark
-   engine.
 
 Building DARS images
-********************
+====================
 
-If you choose to build your own DARS container images, you can customize
-them as needed. Use the provided Dockerfile as a baseline.
+If you choose to build your own DARS container images, you can customize them as needed. Use the :file:`Dockerfile` included in the Github\* repository as your baseline. You can also follow :ref:`custom-app-container` for details on customizing containers on |CL|.
```
Comment (mvincerx) [inline source/guides/stacks/dars.rst]: @bd-dean. I apologize. It looks like your PR will make the finish line before the `custom-app-container.`
So for now, please remove the line :  `You can also follow...`. We can that line back in later on. 
After that's removed, this PR will pass the CI check.

Comment (bd-dean) [reply source/guides/stacks/dars.rst]: backed out, I'll add back with next update

**Comment by mvincerx** [comment]
LGTM. Thanks, @bd-dean.

**Pull request merged**